pkgs/by-name/sl/slackdump/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  buildGo123Module,
```

Comment:
such overrides should be done in all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
as you can see that documentation is for `by-name` packages, this is how `by-name` packages were meant to have overrides